addons/account/models/account_invoice.py

```diff
@@ -1346,6 +1404,27 @@ def _set_additional_fields(self, invoice):
         """
         pass
 
+    edi_description = fields.Char(string='EDI description', compute='_compute_edi_description')
+    edi_product_name = fields.Char(string='EDI product name', compute='_compute_edi_product_name')
+
+    @api.multi
+    @api.depends('name')
+    def _compute_edi_description(self):
+        for record in self:
+            lines = map(lambda line: line.strip(), record.name.split('\n'))
+            record.edi_description = ', '.join(lines)
+
+    @api.multi
+    @api.depends('product_id')
+    def _compute_edi_product_name(self):
+        for record in self:
+            product_id = record.product_id
+            variants = [variant.name for variant in product_id.attribute_value_ids]
+            if variants:
+                record.edi_product_name = '%s (%s)' % (product_id.name, ', '.join(variants))
+            else:
+                record.edi_product_name = product_id.name
```

Comment:
The get_name method is enought I think here, we never used a different way I think